compiler-rt/lib/radsan/radsan_user_interface.cpp

```diff
@@ -17,39 +17,33 @@
 
 namespace radsan {
 
-std::function<OnErrorAction()> createErrorActionGetter() {
-  auto const continue_getter = []() { return OnErrorAction::Continue; };
-  auto const exit_getter = []() { return OnErrorAction::ExitWithFailure; };
-  auto const interactive_getter = []() {
-    auto response = char{};
+bool ShouldExit() {
```

Comment:
Note this function no longer returns a function, but just returns directly whether or not we should exit.